api/shorelineApi.go

```diff
@@ -23,8 +23,10 @@ type (
 		ServerSecret         string `json:"serverSecret"` //used for services
 		LongTermKey          string `json:"longTermKey"`
 		LongTermDaysDuration int    `json:"longTermDaysDuration"`
-		Salt                 string `json:"salt"`      //used for pw
-		Secret               string `json:"apiSecret"` //used for token
+		Salt                 string `json:"salt"`                //used for pw
+		Secret               string `json:"apiSecret"`           //used for token
+		VerificationSkipped  bool   `json:"verificationSkipped"` //allows for the skipping of verification for testing
```

Comment:
Yeah I agree on the name. Initially I had it with just the secret but there was an issue with that approach I am trying to remember